packages/storybook/stories/ovhcloud-design-system/guides/methods.mdx

```diff
@@ -0,0 +1,59 @@
+import { OdsLink } from '@ovhcloud/ods-components/react';
+import { Meta, Source } from '@storybook/blocks';
+import { Heading } from '../../../src/components/heading/Heading';
+
+<Meta title="OVHcloud Design System/Guides/Using Methods" />
+
+<Heading label="Using Methods" level={ 1 } />
+
+Some ODS components are exposing methods.
+
+Those methods behave the same as native Javascript methods (like for example, the `focus()` method), thus they can be used the same way.
+
+Example for an `ods-input`:
+
+<Source code={`
+<ods-input id="my-input" name="my-input">
+</ods-input>
+
+<ods-button id="clear-btn" label="Clear">
+</ods-button>
+
+<script>
+  const inputElement = document.querySelector('#my-input');
+  const clearButton = document.querySelector('#clear-btn');
+
+  clearButton.addEventListener('click', () => {
+    inputElement.clear();
+  });
+</script>
+`}
+        dark="true"
+        language="html" />
+
+<Heading label="React usage" level={ 2 } />
+
+Like native elements, accessing methods need to set a ref to the DOM element (see <OdsLink href="https://react.dev/learn/manipulating-the-dom-with-refs" label="React documentation" target="_blank" />).
+
+Example for an `odsInput`:
+
+<Source code={`
+import { OdsButton, OdsInput } from '@ovhcloud/ods-components/react';
+import { useRef } from 'react';
+
+const MyComponent = () => {
+  const inputRef = useRef<HTMLOdsInputElement>(null);
```

Comment:
Sure the `HTMLOdsInputElement` type can be set here ? No import and this type is generate by stencil & no export by us. Maybe `HTMLElement & OdsInput` should be better